internal/operators/manager_test.go

```diff
@@ -398,13 +414,39 @@ var _ = Describe("Operators manager", func() {
 				{Name: "cnv"},
 			}
 			cluster.CPUArchitecture = common.ARM64CPUArchitecture
-			err := manager.EnsureOperatorArchCapability(cluster, cluster.CPUArchitecture, monitoredOperators)
+			When("OCP verison below 4.14", func() {
```

Comment:
Since the gap is only for CNV I wouldn't duplicate all the operators validations. Here `DescribeTable` is also suitable a will make test test more readable (only cnv should have two entires, one for each version, the rest can have only one entry in the table).